docs/source/users/algorithms/bayesian.rst

```diff
@@ -0,0 +1,104 @@
+
+In addition, if it is assumed that the noise associated with the measured data can be represented as a Gaussian process then
+
+.. math::
+    :label: Gaussian_process
+
+    p(y_i | \boldsymbol{\theta}) = \frac{1}{\sigma_i \sqrt{2 \pi}} \text{exp} \left[ -\frac{(f(\theta, y_i) - y_i)^2}{2 \sigma_i^2} \right].
+
+Equations :math:numref:`independent_data` and :math:numref:`Gaussian_process`, then allow the likelihood function (Equation :math:numref:`likelihood`) to be approximated by
+
+.. math::
+    p(\boldsymbol{y} | \boldsymbol{\theta}) \propto \text{exp} \left( -\frac{\chi^2}{2} \right)
+
+Therefore, the logarithm of the posterior pdf is given by:
+
+.. math::
+    :label: loglike
+
+    \text{log}_e \left[ p(\boldsymbol{\theta} | \boldsymbol{y} )\right] = - \frac{\chi^2}{2}.
+
+More information on Bayesian methods is presented in the book Data Analysis - A Bayesian Tutorial, by D.S.Sivia.
+
+.. _MCMC:
+
+Markov Chain Monte Carlo Methods
+********************************
+
+Markov Chain Monte Carlo (MCMC) is a general method based on drawing values of :math:`\theta` from approximate distributions and then correcting those draws to better approximate the target posterior distribution, :math:`p(\theta|y)`.
+
+A Markov Chain is a sequence of random variables :math:`\theta^1, \theta^2, ...,`` for which, at any t, the distribution of :math:`\theta^t` given all previous :math:`\theta`'s depends only on the most recent value (:math:`\theta^{t-1}`). 
+In MCMC methods, the distribution of the sampled draws depends on the last value drawn, i.e. from a Markov Chain. At each step, the approximate distributions are improved, converging towards a target distribution.
+
+Markov Chain simulation is used when it is not possible to sample directly from :math:`p(\theta|y)`, instead sampling iteratively in such a way that at each step of the process we expect to draw from a distribution that becomes closer to :math:`p(\theta|y)`.
+
+**Metropolis-Hastings Algorithm**
+
+1. Draw starting point :math:`\theta^0` (can be user specified or approximation based on data).
+2. For :math:`t = 1,2,...,T` where T is chain size:
+    a. Sample :math:`\theta^*` from a proposal distribution :math:`J_t(\theta^*|\theta^{t-1})`.
+    b. Calculate the ratio of the target densities evaluated at :math:`\theta^*` and :math:`\theta^{t-1}`:
+            
+        .. math::
+            r=\frac{p(\theta^*|y)/J_t(\theta^*|\theta^{t-1})}{p(\theta^{t-1}|y)/J_t(\theta^{t-1}|\theta^*)}
+    c. Set
+
+        .. math::
+            \theta^t = \begin{cases}\theta^*, & \text{with probability}\ min(r,1) \\ \theta^{t-1}, & \text{otherwise} \end{cases}
+
+Points to consider
+******************
+- MCMC methods do not have an explicit goal of refining the solution, so often a large chain size is required for convergence.
+- It can take some time for the chain to converge to the target distribution and so it's common to discard a number of initial samples
```

Comment:
we dont discard the samples, we keep the same number for the whole simulation. We just dont measure the distribution during the burn in period.